src/jbi/runner.py

```diff
@@ -77,24 +69,24 @@ def execute_action(
             action.whiteboard_tag,
             action.module,
             bug_obj.id,
-            extra={"operation": Operations.EXECUTE, **log_context},
+            extra={"operation": Operation.EXECUTE, **log_context},
         )
 
-        content = action.caller(payload=request)
+        operation, details = action.caller(payload=request)
```

Comment:
Thinking on the naming here... maybe "operation"  should have some implication that it's use is for logs?